content/docs/notebooks/setup.md

```diff
@@ -0,0 +1,251 @@
++++
+title = "Set Up Your Notebooks"
+description = "Getting started with Jupyter notebooks on Kubeflow"
+weight = 10
++++
+
+Your Kubeflow deployment includes services for spawning and managing Jupyter
+notebooks. 
```

Comment:
In this context, the Jupyter Notebook application is being spawned and managed. Should "Jupyter notebooks" be "Jupyter Notebook"? For example:

Your Kubeflow deployment includes support for spawning and managing Jupyter Notebook instances.

or

Your Kubeflow deployment includes support for spawning and managing Jupyter Notebook servers.

Reply:
I'm going to stick with "Jupyter notebooks" for simplicity across the docs, as I think that will be clearer to our readers. It may come across as a little confusing to sometimes use "Notebook" and other times "notebook", though I do take your point about correctness here.